applications/ipfs_utilities_2.md

```diff
@@ -0,0 +1,165 @@
+| ------: | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **0a.** | License                                  | Apache 2.0 / GPLv3 / MIT / Unlicense                                                                                                                                                                                                                                                                                                 |
+| **0b.** | Documentation                            | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works.                                                                                        |
+| **0c.** | Testing and Testing Guide                | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                                                                                                                                      |
+| **0d.** | Docker                                   | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.                                                                                                                                                                                                                        |
+|     0e. | Article                                  | We will write and publish a Medium article to spread the word and give developers an introduction to the project and how to use it.                                                                                                                                                                                                  |
+|      1. | Research                                 | A research needs to take place on how to use InputStreams with RPCs. There are several approaches that need to be checked. In this first part a POC mock implementation will be created.                                                                                                                                             |  |
```

Comment:
Regarding the research deliverable, can you expand on what the PoC/mock implementation will look like? Will we be evaluating the actual research here?

Reply:
@keeganquigley, we updated our proposal regarding your questions. Please let me know if you need anything else.